pkgs/swift2objc/lib/src/transformer/transformers/transform_initializer.dart

```diff
@@ -80,3 +107,37 @@ List<String> _generateInitializerStatements(
     return ['${wrappedClassInstance.name} = $instanceConstruction'];
   }
 }
+
+List<String> _generateMethodStatements(
```

Comment:
This shares a lot of logic with `_generateInitializerStatements`. See if you can factorize them a bit.